tests/python/PyImportSortBearTest.py

```diff
@@ -1,29 +1,93 @@
 from bears.python.PyImportSortBear import PyImportSortBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
-PyImportSortBearTest = verify_local_bear(PyImportSortBear,
-                                         ('import os\nimport sys\n',
-                                          'import os\nimport sys\n'),
-                                         ('import sys\nimport os\n',
-                                          'import sys\nimport os\n'))
-
-PyImportSortBearConfigsTest = verify_local_bear(
-    PyImportSortBear,
-    ('from os import read\nfrom sys import *\n',),
-    ('from os import read\nfrom os import *\n',),
-    settings={'combine_star_imports': True})
```

Comment:
None of the test cases are useful IMO.
They are just testing of the setting is being passed correctly, which is being tested in my test case and as we are moving away from verify_local_bear and since I already have written tests, I didn't want to make the test file bloated. If you want I can revert this change(I am not with this decision though).
Here, the tests are only testing the settings, and having various test cases altering each setting is not quite useful, unless we are testing the tool, which we are not, because it is not our job to test the tool.